Resources/config/typo3.xml

```diff
@@ -102,5 +102,8 @@
             <factory service="Bartacus\Bundle\BartacusBundle\Typo3\ServiceBridge" method="makeInstance"/>
             <argument>TYPO3\CMS\Core\Registry</argument>
         </service>
+
+        <service alias="typo3.connection_pool" id="TYPO3\CMS\Core\Database\ConnectionPool" />
+        <service id="typo3.connection_pool" class="TYPO3\CMS\Core\Database\ConnectionPool" lazy="true" public="true" />
```

Comment:
Remove the service with id `typo3.connection_pool`. Only a service with the FQCN is required. Is there a need for lazy too?

The old id based services are only needed on existing services as BC layer, but deprecated.

Reply:
Indeed. I removed the obsolete code and there was no need for lazy so i removed it too. Please check. Thanks